mcodingbot/plugins/python.py

```diff
@@ -0,0 +1,47 @@
+import re
+
+import crescent
+import hikari
+
+from mcodingbot.utils import Plugin
+
+plugin = Plugin()
+
+PEP_REGEX = re.compile(r"pep *(?P<pep>\d{1,4})", re.IGNORECASE)
+
+
+def get_pep_link(pep_number: int) -> str:
+    return f"<https://peps.python.org/pep-{pep_number:04}/>"
```

Comment:
@ everyone here
Thoughts about letting this embed?

Reply:
wdym

Reply:
Should make it a kwarg tbh. When responding to a message I don't want it because embeds take up a decent amount of space but for the command it seems fine to me.

Reply:
nah, but I was thinking about making the entire reply an embed so you can have hyperlinks when multiple PEPs are mentioned maybe

Reply:
I suggest that we leave it like that for now, we can still change it to fit our convenience later

Reply:
> nah, but I was thinking about making the entire reply an embed so you can have hyperlinks when multiple PEPs are mentioned maybe

We can make that into a new issue so this can be merged and anyone else can do that

Reply:
> We can make that into a new issue so this can be merged and anyone else can do that
This tbh

Reply:
![image](https://user-images.githubusercontent.com/43090614/189745668-11f30fde-c741-4195-9956-083964486915.png)
(on a related note, tbh the link embeds pretty nicely, not that big)

Reply:
I say leave it as-is. Those embeds contain no information related to the PEP, so it's useless. Might make a PR later that scrapes the PEPs for an overview or something, but I would leave it like this for now.

Reply:
> Those embeds contain no information related to the PEP
The title

Reply:
Somehow I missed that smh. Would it be worth making the /pep command embed, but the event hide the embeds?

Reply:
> Somehow I missed that smh. Would it be worth making the /pep command embed, but the event hide the embeds?

I feel like we should just leave that to a new PR.

Reply:
Sounds good to me